extra/release_tool.py

```diff
@@ -2133,6 +2140,68 @@ def do_verify_integration_references(args, optional_too):
         print("\nMake sure all *.yml files contain the correct versions.")
         sys.exit(1)
 
+def is_repo_fresh_master(path):
+    """ Check if we're on the most recent commit in 'master'.
+    """
+    sha_master = execute_git(None, path, ["rev-parse", "origin/master"], capture=True, capture_stderr=True)
```

Comment:
You should not hardcode "origin" here, there are developers (like me) that use a different remote name. See code using `find_upstream_remote(state, repo.git())` for inspiration, for instance [here](https://github.com/mendersoftware/integration/blob/master/extra/release_tool.py#L1431).
```

Reply:
Sorry, published this early by mistake! I will prepare the rest of the comments in one go, you can wait for the rework.

Reply:
fixed, can you check?

Reply:
ps. I might still do some fixes concerning PRs for releases that you mentioned, for now it's a basic fix.

Reply:
:+1: